### PR TITLE
Add music21 to the list of supporting projects

### PR DESCRIPTION
**Pull request by mscuthbert**
We have not decided on a firm date in 2017, but definitely by the end of summer the last Py2 code will be done.

### CODE REVIEW

File: _sections/30-projects.md [hunk 1]
```diff
@@ -27,5 +27,6 @@ style: center
 - [Altair](https://github.com/ellisonbg/altair)
 - [gala](https://gala.readthedocs.io)
 - [cual-id](https://github.com/johnchase/cual-id)
+- [music21](https://web.mit.edu/music21/)
```
Comment (takluyver) [inline _sections/30-projects.md]: I get a 'certificate error' from the https site, but changing it to http works.

Also, we list projects in rough order of prominence, using Github stars as a metric where possible. Looking at [this repo](https://github.com/cuthbertlab/music21), I think you can go just under readiab.

Comment (mscuthbert) [inline _sections/30-projects.md]: Thanks -- will fix both.

**Comment by mscuthbert** [comment]
updated.

**Comment by takluyver** [comment]
Looks good, thanks. :-)

**Comment by mscuthbert** [comment]
We're releasing the beta version of our 4.0 release today -- our last version to support Python 2, which also includes and environmental warning on startup saying that it is the last version to support Python 2 and that users should move to Python 3 immediately.  We'll make the full release in June and then immediately dump Python 2 from the code base, aiming for a Fall Py3 only release (once enough new features are added to convince Py2 people to switch).

**Comment by takluyver** [comment]
Cool, good luck :-)

**Pull request merged**